### PR TITLE
nixos-scripts: init at 0.1

### PR DESCRIPTION
**Pull request by matthiasbeyer**
Packaged my own scripts! :-)

Hopefully they will be usefull...

### CODE REVIEW

File: pkgs/tools/misc/nixos-scripts/default.nix [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+  buildInputs = [ ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp ./nix-* $out/bin/
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Utility scripts for working with nixos tools";
```
Comment (edolstra) [inline pkgs/tools/misc/nixos-scripts/default.nix]: Spelling: "NixOS".

Also, "utility scripts for working with nixos tools" => "Utilities for NixOS".

File: pkgs/tools/misc/nixos-scripts/default.nix [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchurl, mpd_clientlib, curl, glib, pkgconfig }:
+
+stdenv.mkDerivation rec {
+  version = "0.1";
+  name = "nixos-scripts-${version}";
```
Comment (edolstra) [inline pkgs/tools/misc/nixos-scripts/default.nix]: This name suggests that these scripts are part of NixOS, which may be confusing.

Comment (matthiasbeyer) [inline pkgs/tools/misc/nixos-scripts/default.nix]: So what do you suggest?

**Comment by jagajaga** [comment]
I don't see a point for that scripts to be included in nixpkgs repo.. 

May be some information why do we need them? :smile:

**Comment by matthiasbeyer** [comment]
Well, we don't _need_ them. I wouldn't care if this gets rejected,... it's just a proposal!

**Comment by jagajaga** [comment]
You can always manage your own local set of pkgs.
But if this tool is cool for nixos then let's have it.
But I think: maybe let's wait for 1.0 now?

-- 
Sincerely,
Arseniy Seroka

**Comment by matthiasbeyer** [comment]
@jagajaga Okay, I'll release v0.2 hopefully soon ... 1.0 will take a bit longer, but hey... I don't mind!

**Pull request closed without merging**